openAPI/quran.com/v4.json

```diff
@@ -2113,12 +2113,12 @@
         "tags": [
           "Quran"
         ],
-        "description": "Get a single #model:XCinMTfQdCrC3D3vH of all ayah.\n\nSee #endpoint:N4JAF8phDshhyrBHs endpoint to fetch available translations.\n\nYou can also include more fields of translation using `fields` query string.",
+        "description": "Get a single translation of the Qur'an\n\nUse https://api-docs.quran.com/docs/quran.com_versioned/translations to get the ID a desired translation",
```

Comment:
This bit "to get the ID a desired translation" needs rewording to "to get the ID of the desired translation" or "to get the desired translation's ID"